src/gov-action-contracts/AIPs/AIP4844/SetBatchPosterManager.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.16;
+
+import "../../address-registries/L1AddressRegistry.sol";
+import "@openzeppelin/contracts/utils/Address.sol";
+
+interface ISeqInbox {
+    function batchPosterManager() external view returns (address);
+    function setBatchPosterManager(address newBatchPosterManager) external;
+}
+
+/// @notice Sets the batch poster manager role
+/// @dev    This contract is dependent on AIP4844Action having run already, since that action upgrades
+///         the sequencer inbox to the have the batch poster manager role
+///         Identical copies of this contract to be deployed for Arb One and Nova on Ethereum
```

Comment:
I think we should remove this line (with the references to arb1 & nova) to keep this generic and include separate child contracts that set the values in the constructor for arb1 and nova

Reply:
I'm happy to remove the line, but I'm not sure about the child contracts. We wont know about those until we've chosen the batch poster manager address, so we cant create those yet.

Reply:
true